rules/Proj1200.md

```diff
@@ -0,0 +1,35 @@
+# Proj1200: Exclude private assets as project file dependency
+Private assets (such as analyzers) are package references that have an affect
+on compiling a .NET assembly, but are not needed by the compiled artifact. By
+marking them in the project file as private asset it will not become a runtime
+dependency, that linked to a package, or shiped when publishing it.
+
+## Non-compliant
+``` XML
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PackageReference Include="DotNetProjectFile.Analyzers" Version="*" />
+
+</Project>
+```
+
+## Compliant
```

Comment:
What about IncludeAssets and ExcludeAssets? And what about other values apart from "all" ("none", "contentfiles", "analyzers", "build")

Reply:
Theoretically you could have other values, but this rule says that you should use "all".